src/Compilers/CSharp/Portable/FlowAnalysis/PreciseAbstractFlowPass.cs

```diff
@@ -2328,6 +2328,12 @@ public override BoundNode VisitUnboundLambda(UnboundLambda node)
             return VisitLambda(node.BindForErrorRecovery());
         }
 
+
+        public override BoundNode VisitUnboundObjectCreationExpression(UnboundObjectCreationExpression node)
+        {
+            return null;
```

Comment:
>null [](start = 19, length = 4)

I suspect this is not reachable. If so, use `throw ExceptionUtilities.Unreachable;` instead.

Reply:
It is in error cases. I added a comment.